Rewording "Project Properties" to "Properties" and removing icon

DiGro opened the pull request:
Line 54 and 55 :  the icon |option| is not visible in the menu Project next to "Properties" and should be removed
The option for the menu "Project Properties" in Project menu is throughout the application replaced by "Properties" (see also line 116)

### Description

Goal: correct display of items of the GUI in the documentation

- [x] Backport to LTR documentation is required

havatv commented:
The dialogue is still called "Project properties". The menu entry for opening that dialogue is "Properties".

havatv commented on source/docs/user_manual/working_with_projections/working_with_projections.rst:
```diff
@@ -51,8 +51,8 @@ specification of the CRS. This projection file has the same base name as the
-changed in the :guilabel:`General` tab of the |options|
-:guilabel:`Project Properties` dialog under the :guilabel:`Project` menu.
+changed in the :guilabel:`General` tab of the
+:guilabel:`Properties` dialog under the :guilabel:`Project` menu.
```
```suggestion
:guilabel:`Project properties` dialog (:menuselection:`Project --> Properties...`).
```

DiGro replied:
O yeah, I can live with that...

Thanks

havatv replied:
If you agree with my suggestion, you can commit it (I am not allowed to).

DiGro commented:
No problem with the suggestion.
However I've no rights to commit either  ;-)

__________
DiGro 

    Op zaterdag 13 april 2019 22:42:37 CEST schreef Håvard Tveite <notifications@github.com>:  
 
 
@havatv commented on this pull request.

In source/docs/user_manual/working_with_projections/working_with_projections.rst:
> @@ -51,8 +51,8 @@ specification of the CRS. This projection file has the same base name as the
 :file:`alaska.prj`.
 
 Whenever you select a new CRS, the layer units will automatically be
-changed in the :guilabel:`General` tab of the |options|
-:guilabel:`Project Properties` dialog under the :guilabel:`Project` menu.
+changed in the :guilabel:`General` tab of the
+:guilabel:`Properties` dialog under the :guilabel:`Project` menu.

If you agree with my suggestion, you can commit it (I am not allowed to).

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.

DelazJ commented:
> No problem with the suggestion.
However I've no rights to commit either  ;-)

@DiGro as the PR/branch owner you have almost/all rights on it.
To insert @havatv comment, press the "commit suggestion" button and apply it.

DiGro commented:
Done and closed

DelazJ commented:
You don't have to close it. Your commit simply adds Havard suggestion to your changes; it's not yet merged in master. And to do that, you indeed don't have the rights.
